logger.go

```diff
@@ -100,7 +102,21 @@ var LogFormatter = func(values ...interface{}) (messages []interface{}) {
 			messages = append(messages, fmt.Sprintf(" \n\033[36;31m[%v]\033[0m ", strconv.FormatInt(values[5].(int64), 10)+" rows affected or returned "))
 		} else {
 			messages = append(messages, "\033[31;1m")
-			messages = append(messages, values[2:]...)
+
+			var message interface{}
```

Comment:
The entire gorm package is database agnostic (until now) ... 
I suggest you revert your changes here, and instead override the LogFormatter in cfgrevel package.